webpack/components/AnsibleRolesAndVariables/AnsibleRolesAndVariablesHelpers.js

```diff
@@ -0,0 +1,9 @@
+export const prepareResult = rows => {
```

Comment:
Could you add a test please? [Tests for plain js functions](https://github.com/theforeman/foreman/blob/develop/webpack/assets/javascripts/react_app/components/SettingsTable/__tests__/SettingsTableHelpers.test.js#L49-L70) are quite easy to write and do not require snapshots.

Reply:
done